app/scripts/views/confirm.js

```diff
@@ -115,15 +121,22 @@ define(function (require, exports, module) {
               } else {
                 self.navigate('signup_complete');
               }
-            }, function (err) {
```

Comment:
Interesting, it seems to be this path should have been broken before for email bounce.